### PR TITLE
First round of fixes for java related examples

### PR DESCRIPTION
**Pull request by vikramvi**
I have fixed some of the test cases under Jave repo.

Main Changes

1. updated latest Java Client version
2. Removed compilation errors
3. updated element identifiers

Please review changes and merge.

Thanks & Regards,
Vikram

### CODE REVIEW

File: sample-code/examples/java/junit/src/test/java/com/saucelabs/appium/SauceTest.java [hunk 1]
```diff
@@ -69,8 +69,8 @@ public void setUp() throws Exception {
         String sauceUserName = authentication.getUsername();
         String sauceAccessKey = authentication.getAccessKey();
         DesiredCapabilities capabilities = new DesiredCapabilities();
-        capabilities.setCapability("platformVersion", "7.1");
-        capabilities.setCapability("deviceName", "iPhone Simulator");
+        capabilities.setCapability("platformVersion", "9.3");
+        capabilities.setCapability("deviceName", "iPhone6");
```
Comment (imurchie) [inline sample-code/examples/java/junit/src/test/java/com/saucelabs/appium/SauceTest.java]: Does this work? I would have expected `iPhone 6`.

**Comment by imurchie** [comment]
This looks good to me, though I am not a Java expert.

File: sample-code/examples/java/junit/src/test/java/com/saucelabs/appium/SauceTest.java (hunk 1 above)
Comment (vikramvi) [inline sample-code/examples/java/junit/src/test/java/com/saucelabs/appium/SauceTest.java]: @imurchie  Thanks for review and reporting this issue. Have fixed it.

**Comment by SrinivasanTarget** [comment]
@vikramvi Why was this line of code commented https://github.com/vikramvi/sample-code/blob/a107caa8f07be89f532194d8a04d64b45e2c5c85/sample-code/examples/java/junit/src/test/java/com/saucelabs/appium/UICatalogTest.java#L137 ?

Is it throwing any error if so can you fix it instead of commenting it?

**Comment by SrinivasanTarget** [comment]
@vikramvi https://github.com/vikramvi/sample-code/blob/a107caa8f07be89f532194d8a04d64b45e2c5c85/sample-code/examples/java/junit/src/test/java/com/saucelabs/appium/driver_service/DriverServiceSample.java#L55 remove this as well

**Comment by vikramvi** [comment]
@SrinivasanTarget  Thanks for review comments. 

1. will remove testTextFieldEdit() , 

2. didn't understand checkTheAbilityToStartADriverWithTheDefaultServerAndNode() test case ? Can you please clarify , what this is suppose to do ?

**Comment by vikramvi** [comment]
I've removed the test case ( as app behaviour is changed ) and other one will fix during next round of overall fixes.

@TikhomirovSergey @SrinivasanTarget  please recheck and merge.

**Comment by SrinivasanTarget** [comment]
@vikramvi Can you squash your commits?

**Comment by vikramvi** [comment]
@SrinivasanTarget  squashing done.

**Comment by SrinivasanTarget** [comment]
Ok Thanks @vikramvi

**Comment by SrinivasanTarget** [comment]
Merged now @vikramvi

**Comment by vikramvi** [comment]
Thanks a ton @SrinivasanTarget

**Pull request merged**